ebill_postfinance/views/ebill_postfinance_service.xml

```diff
@@ -0,0 +1,136 @@
+            <form>
+                <header>
+                    <button
+                        type="object"
+                        name="test_ping"
+                        string="Test connection"
+                        class="oe_highlight"
+                    />
+                    <button
+                        type="object"
+                        name="search_invoice"
+                        string="Search Invoices"
+                        class="oe_highlight"
+                    />
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="%(contract_for_service)d"
+                            type="action"
+                            class="oe_stat_button"
+                            icon="fa-handshake-o"
+                        >
+                          Contracts
+                        </button>
+                        <button
+                            name="%(invoices_sent_to_service)d"
+                            type="action"
+                            class="oe_stat_button"
+                            icon="fa-envelope-o"
+                        >
+                          Invoices
+                        </button>
+                        <button
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-archive"
+                        >
+                            <field
+                                name="active"
+                                widget="boolean_button"
+                                options="{&quot;terminology&quot;: &quot;disabled&quot;}"
+                            />
+                        </button>
+                    </div>
+                    <label for="name" class="oe_edit_only" />
+                    <h2>
+                        <field name="name" class="oe_inline" />
+                    </h2>
+                    <group>
+                        <group>
+                            <field name="use_test_service" />
+                            <field name="operation_timeout" />
+                            <field name="file_type_to_use" readonly="1" />
+                            <field
+                                name="use_file_type_xml_paynet"
+                                attrs="{'invisible':[('file_type_to_use', '!=', 'XML')]}"
+                            />
+                            <field
+                                name="partner_bank_id"
+                                options="{'no_create': True}"
+                            />
+                        </group>
+                        <group>
+                            <field name="biller_id" />
+                            <field name="username" />
+                            <field name="password" />
```

Comment:
```suggestion
                            <field name="password"  password="True"/>
```
Could you please hide the password value?

Reply:
done